### PR TITLE
W 13734960 2

### PR DESCRIPTION
**Pull request by balbifm**

### CODE REVIEW

**Review by nledesma — APPROVED** [review]

File: Makefile [hunk 1]
```diff
@@ -41,24 +41,14 @@ release: build
...
 	@anypoint-cli-v4 pdk policy-project build-asset-files --version $(ASSET_VERSION) --asset-id $(CRATE_NAME)
 	@cargo anypoint config-gen -p -m $(DEFINITION_GCL_PATH) -o src/generated/config.rs
 
-### -- Setup
-# This private goal is to check both the login and the install of the Cargo anypoint plugin was executed
-.SILENT:
---check-setup:
-	@(cargo +nightly config get registries.anypoint.index -Z unstable-options > /dev/null) || ($(SETUP_ERROR_CMD); exit 1)
-	@(cargo --list | grep anypoint > /dev/null) || ($(SETUP_ERROR_CMD); exit 2)
-
```
Comment (NachoRaik) [inline Makefile:54]: Why did you remove the check?

**Review by NachoRaik — CHANGES_REQUESTED** [review]

File: Makefile (hunk 1 above)
Comment (balbifm) [reply Makefile:54]: Yes, it does not work in Windows. I've created a ticket to re-added in the future

**Review by NachoRaik — APPROVED** [review]
